src/main/java/Warehouse/WarehouseManager/employee/Employee.java

```diff
@@ -46,4 +45,11 @@ public class Employee {
 
     @Temporal(TemporalType.TIMESTAMP)
     private LocalDateTime refreshTokenExpirationDate;
+
+
+    public EmployeeDto toEmployeeDto(){
```

Comment:
Due to the single responsibility principle, I think it would be better to perform the conversion to DTO object in a service layer.